tests/deprecations_ignore.yml

```diff
@@ -349,23 +313,8 @@
 - tests/dag_processing/test_processor.py::TestDagFileProcessor::test_execute_on_failure_callbacks_without_dag
 - tests/dag_processing/test_processor.py::TestDagFileProcessor::test_failure_callbacks_should_not_drop_hostname
 - tests/dag_processing/test_processor.py::TestDagFileProcessor::test_process_file_should_failure_callback
-- tests/decorators/test_python.py::TestAirflowTaskDecorator::test_fail_multiple_outputs_key_type
-- tests/decorators/test_python.py::TestAirflowTaskDecorator::test_fail_multiple_outputs_no_dict
-- tests/decorators/test_python.py::TestAirflowTaskDecorator::test_manual_multiple_outputs_false_with_typings
-- tests/decorators/test_python.py::TestAirflowTaskDecorator::test_multiple_outputs
-- tests/decorators/test_python.py::TestAirflowTaskDecorator::test_multiple_outputs_empty_dict
-- tests/decorators/test_python.py::TestAirflowTaskDecorator::test_multiple_outputs_ignore_typing
-- tests/decorators/test_python.py::TestAirflowTaskDecorator::test_multiple_outputs_return_none
-- tests/decorators/test_python.py::TestAirflowTaskDecorator::test_python_callable_arguments_are_templatized
-- tests/decorators/test_python.py::TestAirflowTaskDecorator::test_python_callable_keyword_arguments_are_templatized
-- tests/decorators/test_python.py::TestAirflowTaskDecorator::test_xcom_arg
+# Inlets/Outlets use TaskInstance.xcom_push() with deprecated arguments in airflow/lineage/__init__.py
```

Comment:
https://github.com/apache/airflow/blob/c6bc0529805be98cffbf336070abee32b93ca39a/airflow/lineage/__init__.py#L84-L92

```console
ERROR    airflow.task:taskinstance.py:2974 Task failed with exception
Traceback (most recent call last):
  File "/opt/airflow/airflow/models/taskinstance.py", line 2550, in _run_raw_task
    self._execute_task_with_callbacks(context, test_mode, session=session)
  File "/opt/airflow/airflow/models/taskinstance.py", line 2756, in _execute_task_with_callbacks
    self.task.post_execute(context=context, result=result)
  File "/opt/airflow/airflow/lineage/__init__.py", line 85, in wrapper
    self.xcom_push(
  File "/opt/airflow/airflow/models/baseoperator.py", line 1590, in xcom_push
    context["ti"].xcom_push(key=key, value=value, execution_date=execution_date)
  File "/opt/airflow/airflow/utils/session.py", line 84, in wrapper
    return func(*args, session=session, **kwargs)
  File "/opt/airflow/airflow/models/taskinstance.py", line 3270, in xcom_push
    warnings.warn(message, RemovedInAirflow3Warning, stacklevel=3)
airflow.exceptions.RemovedInAirflow3Warning: Passing 'execution_date' to 'TaskInstance.xcom_push()' is deprecated.
```

Reply:
Why not fix these then? They seem pretty trivial.

Reply:
I want to keep changes related to tests only and core as a separate PRs

Reply:
Solved by: https://github.com/apache/airflow/pull/39327